tests/unit/cluster/atomic-slot-migration.tcl

```diff
@@ -1584,54 +1608,16 @@ start_cluster 3 3 {tags {external:skip cluster} overrides {cluster-node-timeout
         R 0 config set rdb-key-save-delay 0
     }
 
-    test "Test active-trim clears partially imported keys on failover" {
```

Comment:
why did delete this test

Reply:
It seemed duplicate. We don't have anything specific to active trim on a failover scenario. There is one test for trimming on the next ASM operation (uses bgtrim).